zadanie_8_5/zadanie_8_5.js

```diff
@@ -0,0 +1,14 @@
+var a = prompt("Podaj liczbę a: ", 0),
+	b = prompt("Podaj liczbę b: ", 0),
+	value = (a*a) + (2*a*b) - (b*b);
+
+if (value > 0) {
+	console.log('wynik dodatni');
+	document.write('wynik dodatni');
+} else if(value == 0) {
+	console.log('wynik zerowy');
+	document.write('wynik zerowy');
+} else {
+	console.log('wynik dodatni');
+	document.write('wynik ujemny');
+}
```

Comment:
Sugeruje pewne udoskonalenie. Teraz jak wpisze niepoprawne dane np litery to otrzymuje jakis wynik, a chcialbym dostawac informacje "wynik nieokreslony"